### ISSUE TITLE
tensorRT 7.0.0.11 sample ssd error

### ISSUE THREAD

**Issue by coolbei** [NVIDIA/TensorRT#1007]
## Description

upgrade trt from 5.1.5 to 7.0.0.11, sample_ssd not work.

## Environment

**TensorRT Version**: tensorRT 7.0.0.11
**GPU Type**: P100
**Nvidia Driver Version**: 418.67
**CUDA Version**: 10.0
**CUDNN Version**: libcudnn.so.7.6.5
**Operating System + Version**: centos 7.2
**Python Version (if applicable)**: 
**TensorFlow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if container which image + tag)**: 

## Relevant Files

everything is according to SampleSSD and README, deploy.prototxt, ssd.prototxt

## Steps To Reproduce

run the sample

error info:
$ LD_LIBRARY_PATH=~/git/tensorRT/build/out/:~/git/tensorRT/trt_release/lib/ ./sample_ssd
&&&& RUNNING TensorRT.sample_ssd # ./sample_ssd
[01/12/2021-16:06:39] [I] Building and running a GPU inference engine for SSD
[01/12/2021-16:06:43] [I] [TRT] Some tactics do not have sufficient workspace memory to run. Increasing workspace size may increase performance, please check verbose output.

[01/12/2021-16:07:02] [I] [TRT] Detected 1 inputs and 2 output network tensors.
Cuda error in file src/implicit_gemm.cu at line 648: invalid device function
[01/12/2021-16:07:02] [W] [TRT] Current optimization profile is: 0. Please ensure there are no enqueued operations pending in this context prior to switching profiles
#assertion ~/git/tensorRT/plugin/normalizePlugin/normalizePlugin.cpp,115
已放弃

I also try trtexec .     build with a cuda error, and infer with a assert error same as up.
part of the output:

[01/12/2021-17:27:03] [V] [TRT] Tactic: 0 time 0.020064
[01/12/2021-17:27:03] [V] [TRT] Fastest Tactic: 0 Time: 0.020064
[01/12/2021-17:27:03] [V] [TRT] >>>>>>>>>>>>>>> Chose Runner Type: ExtSoftMax Tactic: 0
[01/12/2021-17:27:03] [V] [TRT]
[01/12/2021-17:27:03] [V] [TRT] *************** Autotuning format combination: Float(1,21,183372) -> Float(1,1,1,183372) ***************
[01/12/2021-17:27:03] [V] [TRT] --------------- Timing Runner: mbox_conf_flatten (Shuffle)
[01/12/2021-17:27:03] [V] [TRT] Tactic: 0 is the only option, timing skipped
[01/12/2021-17:27:03] [V] [TRT] Fastest Tactic: 0 Time: 0
[01/12/2021-17:27:03] [V] [TRT] --------------- Timing Runner: <reformat> (Reformat)
[01/12/2021-17:27:03] [V] [TRT] Tactic: 0 time 0.006016
[01/12/2021-17:27:03] [V] [TRT] Fastest Tactic: 0 Time: 0.006016
[01/12/2021-17:27:03] [V] [TRT] *************** Autotuning format combination: Float(1,1,1,34928), Float(1,1,1,183372), Float(1,1,34928,69856) -> Float(1,7,1400,1400), Float(1,1,1,1) ***************
[01/12/2021-17:27:03] [V] [TRT] Formats and tactics selection completed in 16.0266 seconds.
[01/12/2021-17:27:03] [V] [TRT] After reformat layers: 63 layers
[01/12/2021-17:27:03] [V] [TRT] Block size 23040000
[01/12/2021-17:27:03] [V] [TRT] Block size 23040000
[01/12/2021-17:27:03] [V] [TRT] Block size 16777216
[01/12/2021-17:27:03] [V] [TRT] Block size 5760000
[01/12/2021-17:27:03] [V] [TRT] Block size 739328
[01/12/2021-17:27:03] [V] [TRT] Block size 204800
[01/12/2021-17:27:03] [V] [TRT] Block size 184832
[01/12/2021-17:27:03] [V] [TRT] Block size 139776
[01/12/2021-17:27:03] [V] [TRT] Block size 25600
[01/12/2021-17:27:03] [V] [TRT] Block size 9216
[01/12/2021-17:27:03] [V] [TRT] Block size 1024
[01/12/2021-17:27:03] [V] [TRT] Total Activation Memory: 69921792
[01/12/2021-17:27:03] [I] [TRT] Detected 1 inputs and 2 output network tensors.
[01/12/2021-17:27:03] [V] [TRT] conv1_2 + relu1_2 (scudnn_winograd) Set Tactic Name: maxwell_scudnn_winograd_128x128_ldg1_ldg4_relu_tile148m_nt_v1
[01/12/2021-17:27:03] [V] [TRT] fc6 + relu6 (scudnn) Set Tactic Name: maxwell_scudnn_128x64_relu_medium_nn_v1
[01/12/2021-17:27:03] [V] [TRT] fc7 + relu7 (scudnn) Set Tactic Name: maxwell_scudnn_128x32_relu_interior_nn_v1
[01/12/2021-17:27:03] [V] [TRT] conv6_2 + conv6_2_relu (scudnn) Set Tactic Name: maxwell_scudnn_128x32_relu_small_nn_v1
[01/12/2021-17:27:03] [V] [TRT] conv7_2 + conv7_2_relu (scudnn) Set Tactic Name: maxwell_scudnn_128x32_relu_small_nn_v1
Cuda error in file src/implicit_gemm.cu at line 648: invalid device function
[01/12/2021-17:27:03] [V] [TRT] Engine generation completed in 17.5188 seconds.
[01/12/2021-17:27:03] [V] [TRT] Engine Layer Information:
[01/12/2021-17:27:03] [V] [TRT] Layer(CustomImplicitGemmReLU): conv1_1 + relu1_1, Tactic: 0, data[Float(3,300,300)] -> conv1_1[Float(64,300,300)]
[01/12/2021-17:27:03] [V] [TRT] Layer(scudnn_winograd): conv1_2 + relu1_2, Tactic: 5921334924264294896, conv1_1[Float(64,300,300)] -> conv1_2[Float(64,300,300)]
[01/12/2021-17:27:03] [V] [TRT] Layer(PoolingTiled): pool1, Tactic: 5701889, conv1_2[Float(64,300,300)] -> pool1[Float(64,150,150)]
[01/12/2021-17:27:03] [V] [TRT] Layer(CustomWinogradReLu): conv2_1 + relu2_1, Tactic: 1, pool1[Float(64,150,150)] -> conv2_1[Float(128,150,150)]
[01/12/2021-17:27:03] [V] [TRT] Layer(CustomWinogradReLu): conv2_2 + relu2_2, Tactic: 1, conv2_1[Float(128,150,150)] -> conv2_2[Float(128,150,150)]
[01/12/2021-17:27:03] [V] [TRT] Layer(Pooling): pool2, Tactic: -1, conv2_2[Float(128,150,150)] -> pool2[Float(128,75,75)]
[01/12/2021-17:27:03] [V] [TRT] Layer(CustomWinogradReLu): conv3_1 + relu3_1, Tactic: 1, pool2[Float(128,75,75)] -> conv3_1[Float(256,75,75)]
[01/12/2021-17:27:03] [V] [TRT] Layer(CustomWinogradReLu): conv3_2 + relu3_2, Tactic: 1, conv3_1[Float(256,75,75)] -> conv3_2[Float(256,75,75)]
[01/12/2021-17:27:03] [V] [TRT] Layer(CustomWinogradReLu): conv3_3 + relu3_3, Tactic: 1, conv3_2[Float(256,75,75)] -> conv3_3[Float(256,75,75)]
[01/12/2021-17:27:03] [V] [TRT] Layer(PoolingTiled): pool3, Tactic: 6226177, conv3_3[Float(256,75,75)] -> pool3[Float(256,38,38)]
[01/12/2021-17:27:03] [V] [TRT] Layer(CustomWinogradReLu): conv4_1 + relu4_1, Tactic: 1, pool3[Float(256,38,38)] -> conv4_1[Float(512,38,38)]
[01/12/2021-17:27:03] [V] [TRT] Layer(CustomWinogradReLu): conv4_2 + relu4_2, Tactic: 1, conv4_1[Float(512,38,38)] -> conv4_2[Float(512,38,38)]
[01/12/2021-17:27:03] [V] [TRT] Layer(CustomWinogradReLu): conv4_3 + relu4_3, Tactic: 1, conv4_2[Float(512,38,38)] -> conv4_3[Float(512,38,38)]
[01/12/2021-17:27:03] [V] [TRT] Layer(Pooling): pool4, Tactic: -1, conv4_3[Float(512,38,38)] -> pool4[Float(512,19,19)]
[01/12/2021-17:27:03] [V] [TRT] Layer(CustomWinogradReLu): conv5_1 + relu5_1, Tactic: 1, pool4[Float(512,19,19)] -> conv5_1[Float(512,19,19)]

....

[01/12/2021-17:27:03] [V] [TRT] Layer(Reformat): conv7_2_mbox_priorbox copy, Tactic: 0, conv7_2_mbox_priorbox[Float(2,600,1)] -> mbox_priorbox[Float(2,600,1)]
[01/12/2021-17:27:03] [V] [TRT] Layer(Reformat): conv8_2_mbox_priorbox copy, Tactic: 0, conv8_2_mbox_priorbox[Float(2,144,1)] -> mbox_priorbox[Float(2,144,1)]
[01/12/2021-17:27:03] [V] [TRT] Layer(Reformat): conv9_2_mbox_priorbox copy, Tactic: 0, conv9_2_mbox_priorbox[Float(2,16,1)] -> mbox_priorbox[Float(2,16,1)]
[01/12/2021-17:27:03] [V] [TRT] Layer(ExtSoftMax): mbox_conf_softmax, Tactic: 0, mbox_conf_reshape[Float(8732,21)] -> mbox_conf_softmax[Float(8732,21)]
[01/12/2021-17:27:03] [V] [TRT] Layer(PluginV2): detection_out, Tactic: 0, mbox_loc[Float(34928,1,1)], mbox_conf_flatten[Float(183372,1,1)], mbox_priorbox[Float(2,34928,1)] -> detection_out[Float(1,200,7)], keep_count[Float(1,1,1)]

[01/12/2021-17:27:11] [W] set up infer
[01/12/2021-17:27:11] [W] [TRT] Current optimization profile is: 0. Please ensure there are no enqueued operations pending in this context prior to switching profiles
[01/12/2021-17:27:11] [W] check binding
fill --------------------------------------------------------- buffer
[01/12/2021-17:27:11] [W] check binding
[01/12/2021-17:27:11] [W] check binding
#assertion/home/chengxin.jb/git/tensorRT/plugin/normalizePlugin/normalizePlugin.cpp,115
已放弃

**It works on 5.1.5.**

We got tens of hundred of servers, so we can't just upgrade the gpu driver. So the latest release of TRT won't do. TRT7.0.0.11 is the version we want to use.  And we found some bugs of this version which is really frustrating.

**Comment by coolbei** [comment]
My fault.    I have several cuda installed on my server. So I remove cuda10.2 and rebuild. It works.
